pkg/handler/project/project.go

```diff
@@ -325,6 +325,98 @@ func (h *handler) GetMembers(c *gin.Context) {
 		&view.PaginationResponse{Pagination: query.Pagination, Total: total}, nil, nil))
 }
 
+// DeleteMember godoc
+// @Summary Delete member in a project
+// @Description Delete member in a project
+// @Tags Project
+// @Accept  json
+// @Produce  json
+// @Param Authorization header string true "jwt token"
+// @Param id path string true "project ID"
+// @Param member_id path string true "employee ID"
+// @Param Body body DeleteMemberInput true "Body"
+// @Success 200 {object} view.MessageResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /project/:id/members/:member_id [delete]
+func (h *handler) DeleteMember(c *gin.Context) {
+	projectID := c.Param("id")
+	memberID := c.Param("member_id")
+	if projectID == "" {
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, errors.New("invalid project_id"), nil))
+		return
+	}
+
+	// TODO: can we move this to middleware ?
+	l := h.logger.Fields(logger.Fields{
+		"handler": "project",
+		"method":  "DeleteMember",
+		"body":    memberID,
+	})
+
+	// get member infor
```

Comment:
```suggestion
	// get member info
```
but should be deleted

Reply:
updated